alfalfa_worker/worker.py

```diff
@@ -86,17 +87,25 @@ def check_step_sim_config(self, message_body):
         year = datetime.today().year
         start = "{}-01-01 00:00:00".format(year)
         end = "{}-12-31 23:59:00".format(year)
-        start_datetime = message_body.get('startDatetime',
-                                          start)  # TODO change server side message: startDatetime to start_datetime
-        end_datetime = message_body.get('endDatetime',
-                                        end)  # TODO change server side message: endDatetime to end_datetime
+        # TODO change server side message: startDatetime to start_datetime
+        start_datetime = message_body.get('startDatetime', start)
+        if start_datetime == 'undefined':
+            start_datetime = start
+        # TODO change server side message: endDatetime to end_datetime
+        end_datetime = message_body.get('endDatetime', end)
```

Comment:
datetime is weird. FMU wants seconds from 0. I think we should always have datetime as datetime objects, then convert to how OS or FMU wants at time of their execution.

Reply:
Nick - it looks like this PR still keeps it as a string.  Do you want to just open this as an issue to track it?

Reply:
yeah, i kept it as a string. Making this an issue would be the best solution for now.